tests/src/test/spark330/scala/org/apache/spark/sql/rapids/utils/RapidsTestSettings.scala

```diff
@@ -77,7 +77,7 @@ class RapidsTestSettings extends BackendTestSettings {
   enableSuite[RapidsParquetFieldIdSchemaSuite]
   enableSuite[RapidsParquetInteroperabilitySuite]
     .exclude("SPARK-36803: parquet files with legacy mode and schema evolution", KNOWN_ISSUE("https://github.com/NVIDIA/spark-rapids/issues/11454"))
-    .exclude("parquet timestamp conversion", KNOWN_ISSUE("https://github.com/NVIDIA/spark-rapids/issues/11448"))
+    .exclude("parquet timestamp conversion", ADJUST_UT("impala_timestamp cannot be found"))
```

Comment:
So just to be clear her the ADJUST_UT means that the test cannot pass so we wrote a new test to cover the same thing?

Reply:
Yes, the old one need to be exclued from the suites ParquetInteroperabilitySuite which RapidsParquetInteroperabilitySuite inherites. In this case, the impala_timestamp file cannot be found by the way used in the old test case, so I changed some code to enable impala_timestamp to be found and put it in ParquetInteroperabilitySuite.